packages/dd-trace/test/appsec/iast/taint-tracking/sources/taint-tracking.express.plugin.spec.js

```diff
@@ -208,12 +208,7 @@ describe('Path params sourcing with express', () => {
         res.status(200).send()
```

Comment:
this test is now identical to the one above no ?

Reply:
The last one make sense only for express v4